GVFS/GVFS.Common/LocalUpgraderServices.cs

```diff
@@ -0,0 +1,152 @@
+﻿using GVFS.Common.FileSystem;
+using GVFS.Common.Tracing;
+using System;
+using System.IO;
+
+namespace GVFS.Common
+{
+    public class LocalUpgraderServices

```

Comment:
Naming suggestion:  From the name of this class it wasn't clear to me if `LocalUpgraderServices` was related to Windows services.

Maybe something like `CommonUpgraderTasks`?  Or move the code to a common base class?

Reply:
I will think about the name some more - Instead of a "Common" class, I wanted a class that provided all of the local operations that upgraders depend on... the idea with the current name was that it provided local / client  "services" that upgraders use.